Home_Work_11/task2.py

```diff
@@ -0,0 +1,40 @@
+# 2) Создайте класс Архив, который хранит пару свойств. Например, число и строку.
+# При нового экземпляра класса, старые данные из ранее созданных экземпляров сохраняются
+# в пару списков-архивов, которые также являются свойствами экземпляра.
+
+# 3) Добавьте к задачам 1 и 2 строки документации для классов.
+
+# 4) Доработаем класс Архив из задачи 2. Добавьте методы представления экземпляра
+# для программиста и для пользователя.
+
+# Добавьте ко всем задачам с семинара строки документации и методы вывода информации на печать.
+
+class Archive:
+    """The Archive training class."""
+
+    my_list_number = []
+    my_list_string = []
+
+    def __init__(self, number, my_string):
+        """Added number and my_string parameters. Append values in lists my_list_number and my_list_string."""
+        self.number = number
+        self.my_string = my_string
+        self.my_list_number.append(number)
+        self.my_list_string.append(my_string)
```

Comment:
Nne можно напрямую по имени класса обращаться, ведь это же все-таки свойства класса, а не только конкретного объекта
```suggestion
        Archive.my_list_number.append(number)
        Archive.my_list_string.append(my_string)
```